load-generator-source/src/main/resources/config/spring-module.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:int="http://www.springframework.org/schema/integration"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+		http://www.springframework.org/schema/beans/spring-beans.xsd
+		http://www.springframework.org/schema/integration
+		http://www.springframework.org/schema/integration/spring-integration.xsd">
+
+    <bean class="org.springframework.xd.loadgenerator.LoadGenerator">
+        <constructor-arg name="messageCount" value="${messageCount}"/>
+        <constructor-arg name="producers" value="${producers}"/>
+        <constructor-arg name="messageSize" value="${messageSize}"/>
+        <property name="autoStartup" value="false"/>
+        <property name="outputChannel" ref="output"/>
+    </bean>
+
+    <int:channel id="output">
+        <int:dispatcher task-executor="taskExecutor"/>
+    </int:channel>
+
+    <bean id="taskExecutor"
```

Comment:
I had thought that the ${producers} would be a property placeholder for the 'corePoolSize' in the taskExecutor.  
```
<bean id="taskExecutor" class="org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor">
    <property name="corePoolSize" value="${producers}" />
    <property name="maxPoolSize" value="${producers}" />
    <property name="queueCapacity" value="25" />
</bean>
```
However, I think this might not be appropriate, having the thread pool should live inside the module means there is one less thread running overall and it mimics more the case of what would happen in a 'real' source such as rabbit message listener container etc.

That said, the raw thread API shouldn't be used but the Executor API - also cleanup should happen in doStop.  This is a good template to follow, except use Executors.newFixedThreadPool.